scripts/prebuild.js

```diff
@@ -132,11 +133,29 @@ const setVersion = async (buildType, tagName) => {
   fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, undefined, 2));
 };
 
+const setDependencies = () => {
+  const packageJsonPath = path.join(__dirname, "..", "cysync", "package.json");
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath));
+
```

Comment:
There's no need to even read the file when the platform is not `darwin`.

Reply:
Yeah, thats right. Moved it inside the if condition 😅